airbyte-integrations/connectors/source-pipedrive/acceptance-test-config.yml

```diff
@@ -1,72 +1,45 @@
-connector_image: airbyte/source-pipedrive:dev
-test_strictness_level: "high"
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-pipedrive:2.0.0
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_pipedrive/spec.json"
+      - spec_path: "source_pipedrive/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: 1.0.0
   connection:
     tests:
       - config_path: "secrets/config.json"
         status: "succeed"
-      - config_path: "secrets/old_config.json"
-        status: "succeed"
       - config_path: "integration_tests/invalid_config.json"
         status: "failed"
   discovery:
     tests:
       - config_path: "secrets/config.json"
         backward_compatibility_tests_config:
-          disable_for_version: 0.1.19
+          disable_for_version: 1.0.0
```

Comment:
There are not reason to change to breaking change.

Reply:
It shouldn't be the case. For migrating a Beta/GA the schema should be broke right now.

Reply:
Should I remove disable for version

Reply:
Yes, please remove those lines.